index.js

```diff
@@ -27,7 +27,7 @@ exports.watch = function(src, dist, verbose) {
 
                 if (fs.existsSync(path + ".js")) fs.unlink(path + ".js", () => {});
```

Comment:
Is there a specific reason we would want to keep this and .d.ts async?

Reply:
good catch - fixed.